src/mpags-cipher.cpp

```diff
@@ -86,70 +62,58 @@ int main(int argc, char* argv[])
+    if (!inputFile.empty()) {   //if input file is not empty on cmd line arguments
+        std::ifstream in_file{inputFile};   //setup an ifstream of the inputfile name
+        bool readinok = in_file.good(); //setup bool to check if readin is ok
+        if(readinok){   //if readin is true -- i.e. we can readin from the file supplied to the cmd line 
+            in_file >> readinText;  //store input from file in readinText
+            for(std::size_t j{0}; j < readinText.length(); ++j){    //loop over each character in the string
+                inputChar = readinText[j];  //assign the character to inputChar
+                inputText += transformChar(inputChar);  //append to inputText
+            }
+            if(encrypt or decrypt) {    //if encryption or decryption is specified act on this
+                inputText = runCaesarCipher(inputText,key,encrypt,decrypt); //apply encryption or decryption to inputText
+            }
+            inputText += "\n";  //add \n to end of inputText to start a new line in the console and output file
+        } else {    
+            std::cerr << "[Warning] cannot read in from file ('" << inputFile   //if the file cannot be read from then output no zero return
+                      << "')\n";
+            return 1;
         }
-
-        // Transliterate digits to English words
-        switch (inputChar) {
-            case '0':
-                inputText += "ZERO";
-                break;
-            case '1':
-                inputText += "ONE";
-                break;
-            case '2':
-                inputText += "TWO";
-                break;
-            case '3':
-                inputText += "THREE";
-                break;
-            case '4':
-                inputText += "FOUR";
-                break;
-            case '5':
-                inputText += "FIVE";
-                break;
-            case '6':
-                inputText += "SIX";
-                break;
-            case '7':
-                inputText += "SEVEN";
-                break;
-            case '8':
-                inputText += "EIGHT";
-                break;
-            case '9':
-                inputText += "NINE";
-                break;
+    } else {    //if there is no input file then take input from terminal
+        // loop over each character from user input
+        std::cout << "No Input File selected, please input to terminal: " << std::endl; //tells you that you didn't input a file so must input on terminal
+        while (std::cin >> inputChar) {
+            // Uppercase alphabetic characters
+            inputText += transformChar(inputChar);  //transforms text to uppercase and makes numbers into letters (spelt out name)
+            // If the character isn't alphabetic or numeric, DONT add it
+        } 
+        if(encrypt or decrypt) {    //same as above run encryption or decyrption if specified 
+            inputText = runCaesarCipher(inputText,key,encrypt,decrypt);
```

Comment:
This can be taken out of this `if` statement as it's duplicated above. In other words, this code will be run whether using a file or `cin` so shouldn't be copied and pasted in the conditional.